cloud/pkg/taskmanager/util/controller/controller.go

```diff
@@ -38,10 +39,11 @@ type Controller interface {
 	Start() error
 	ReportNodeStatus(string, string, fsm.Event) (api.State, error)
 	ReportTaskStatus(string, fsm.Event) (api.State, error)
-	ValidateNode(util.TaskMessage) []v1.Node
+	ValidateNode(util.TaskMessage) ([]*v1.Node, []*v1.Node)
 	GetNodeStatus(string) ([]v1alpha1.TaskStatus, error)
 	UpdateNodeStatus(string, []v1alpha1.TaskStatus) error
 	StageCompleted(taskID string, state api.State) bool
+	GetTaskNodes(name string) ([]string, error)
```

Comment:
Sorry, I'm still confused about this function `GetTaskNodes`. I cannot find where implement it or use it. Could you please explain more about this?

Reply:
sorry,it is my problem,I forget to delete it .I must apologize for any inconvenience it may have caused you. Thanks for your time.Now is update.